src/zampy/datasets/cds_utils.py

```diff
@@ -0,0 +1,361 @@
+        year_month_pairs, variables, position=0, leave=True
+    ):
+        # raise download request
+        r = client.retrieve(
+            dataset,
+            {
+                "product_type": "reanalysis",
+                "variable": [cds_var_names[variable]],
+                "year": year,
+                "month": month,
+                # fmt: off
+                "day": [
+                    "01", "02", "03", "04", "05", "06", "07", "08", "09", "10",
+                    "11", "12", "13", "14", "15", "16", "17", "18", "19", "20",
+                    "21", "22", "23", "24", "25", "26", "27", "28", "29", "30",
+                    "31",
+                ],
+                "time": [
+                    "00:00", "01:00", "02:00", "03:00", "04:00", "05:00", "06:00",
+                    "07:00", "08:00", "09:00", "10:00", "11:00", "12:00", "13:00",
+                    "14:00", "15:00", "16:00", "17:00", "18:00", "19:00", "20:00",
+                    "21:00", "22:00", "23:00",
+                ],
+                # fmt: on
+                "area": [
+                    spatial_bounds.north,
+                    spatial_bounds.west,
+                    spatial_bounds.south,
+                    spatial_bounds.east,
+                ],
+                "format": "netcdf",
+            },
+        )
+        fpath = path / f"{fname}_{variable}_{year}-{month}.nc"
+        _check_and_download(r, fpath, overwrite)
+
+
+def retrieve_cams(
+    client: cdsapi.Client,
+    fname: str,
+    dataset: str,
+    variables: list[str],
+    time_bounds: TimeBounds,
+    spatial_bounds: SpatialBounds,
+    path: Path,
+    cds_var_names: dict[str, str],
+    overwrite: bool,
+) -> None:
+    """Download data via ADS API.
+
+    Args:
+        client: CDS API client.
+        fname: Dataset name alias.
+        dataset: Dataset name for retrieval via `cdsapi`.
+        variables: Zampy variables.
+        time_bounds: Zampy time bounds object.
+        spatial_bounds: Zampy spatial bounds object.
+        path: File path to which the data should be saved.
+        cds_var_names: Variable names from CDS server side.
+        overwrite: If an existing file (of the same size!) should be overwritten.
+    """
+    # make sure time format is YY-MM-DD
+    time_start = str(np.datetime_as_string(time_bounds.start, unit="D"))  # please mypy
+    time_end = str(np.datetime_as_string(time_bounds.end, unit="D"))
+    # raise download request
+    for variable in tqdm(variables):
+        r = client.retrieve(
+            dataset,
+            {
+                "model_level": "60",
+                "variable": [cds_var_names[variable]],
+                "date": f"{time_start}/{time_end}",
+                "step": ["0", "3", "6", "9", "12", "15", "18", "21"],
+                "area": [
+                    spatial_bounds.north,
+                    spatial_bounds.west,
+                    spatial_bounds.south,
+                    spatial_bounds.east,
```

Comment:
We need to say in the documentation that `model_level` is set to 60 and all steps are downloaded.